experiments/long_runs/land_region.jl

```diff
@@ -374,6 +374,7 @@ function setup_prob(t0, tf, Δt; outdir = outdir, nelements = (10, 10, 15))
         subsurface_space,
         outdir;
         start_date,
+        num_points = n_elements,
```

Comment:
why does this one use `n_elements`?

Reply:
this one is regional! so I thought it might make the most sense to use n_elements (nx,ny,nz) directly. For global runs, we specify n_elements = (n_horizontal, nz). But, we can also make the global runs use something based on n_elements (and n_polynomial)

Reply:
right, good point!

Reply:
I'm okay with hardcoding the points for the global runs since we rarely change the resolution, but it would be good to add in-line comments explaining that with these values we get 1:1 output to simulation points